package.json

```diff
@@ -11,7 +11,7 @@
     "client"
   ],
   "engines": {
-    "node": ">= 6.11.5"
+    "node": ">= 8.16.0"
```

Comment:
Why `8.16.0`? We should support with `8.9.0` (low LTS verion)

Reply:
updated